### PR TITLE
[DOC-3331] SMP 0.7.0 79421 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.7.0 79421 release notes

Drafted from the lists provided by Faruk:
https://harness.atlassian.net/wiki/spaces/PD/pages/21427290141/Release+Notes+Current+Gen-+06+30+2023
https://harness.atlassian.net/wiki/spaces/PD/pages/21427290162/Release+Notes+Next+Gen-+06+30+2023

Here is a list of items that are not included in the draft. These items were included on the above lists but were not published in module SaaS release notes. We may need to update our filter to include only items with "`Found In `= `PROD`".
https://docs.google.com/document/d/1vhxELtHY5WwX81q_KanYQC0U_x-bq5YLN8Idf4e6SyU/edit

For reviewers, previews are available:
[FG](https://649df92e449f80377f7b81b1--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes)
[NG](https://649f234a967a46305e58ec7c--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition)

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
Need to add release notes for - [SMP-1352 Helm chart changes for external postgres](https://harness.atlassian.net/browse/SMP-1352?atlOrigin=eyJpIjoiZTI1MzMyMTJhZmZiNDQ5NzhmOWU4OWIwNWZhODliZGIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

**Review by pratmit — APPROVED** [review]
Conditionally approve but the SMP release notes process need to be revamped, the engineering signoffs will be improved which will help the release notes.

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -23,6 +23,455 @@ Review the notes below for details about recent changes to Harness Self-Managed
...
+  
+  Now both deployment types include [infrastructure provisioning steps](https://developer.harness.io/docs/category/provision-infrastructure).
+
+- Bamboo triggers were not working properly. (CDS-69605)
+  
+  Adding the Bamboo build to the delegate response resolved this issue. 
+
+- Selecting stages in Triggers was resetting to all stages in the YAML editor. (CDS-69725)
+  
+  Now Harness avoids sending multiple API calls for merge, and stage selecting works as intended.
```
Comment (StrawberryPad) [inline release-notes/self-managed-enterprise-edition.md:364]:
```suggestion
  Now Harness avoids sending multiple API calls for merge, and selecting stages works as intended.
```

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -23,6 +23,455 @@ Review the notes below for details about recent changes to Harness Self-Managed
...
+
+#### Harness Platform
+
+- The secrets list API return a status code of `200` when permissions are missing. (PL-26474)
+
+  The API now returns a status code of `403` when permissions are missing on secrets.
+
+- The error message displayed when permissions are missing during pipeline execution does not mention the corresponding resource Id. (PL-31350)
+
+  A code enhancement to display the resource Id in the error message fixed this issue.
```
Comment (StrawberryPad) [inline release-notes/self-managed-enterprise-edition.md:376]:
```suggestion
  A code enhancement to display the resource ID in the error message fixed this issue.
```

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -23,6 +23,455 @@ Review the notes below for details about recent changes to Harness Self-Managed
...
+
+- Invites to users fail with an unauthorized error while RBAC setup is still in progress. (PL-32117)
+
+  A polling system ensures that RBAC setup has been completed. The Harness Delegate version 79307 is required for this fix.
+
+- The comparison of email addresses during sign in is case-sensitive. (PL-32198)
```
Comment (StrawberryPad) [inline release-notes/self-managed-enterprise-edition.md:382]:
```suggestion
- The comparison of email addresses during sign-in is case-sensitive. (PL-32198)
```

**Review by StrawberryPad — DISMISSED** [review]

**Review by pratmit — APPROVED** [review]

**Comment by swapnil-harness** [comment]
@brian-f-harness For accesscontrol-service-signed image we have 79004 can you please update the release notes

**Comment by brian-f-harness** [comment]
> @brian-f-harness For accesscontrol-service-signed image we have 79004 can you please update the release notes

@swapnil-harness updated. Thank you.

**Review by aimurphy — APPROVED** [review]
approving known issue and airgap link

**Review by swapnil-harness — APPROVED** [review]
LGTM

**Pull request merged**